pkg/hive/install.go

```diff
@@ -195,6 +202,7 @@ func (c *clusterManager) clusterDeploymentForInstall(doc *api.OpenShiftClusterDo
 			},
 			ControlPlaneConfig: hivev1.ControlPlaneConfigSpec{
 				APIServerIPOverride: doc.OpenShiftCluster.Properties.NetworkProfile.APIServerPrivateEndpointIP,
+				APIURLOverride:      fmt.Sprintf("api-int.%s:6443", clusterDomain),
```

Comment:
might be overkill, but
```suggestion
				APIURLOverride:      net.JoinHostPort("api-int." + clusterDomain, "6443"),
```